protocol/rpcprovider/rpcprovider.go

```diff
@@ -307,12 +310,16 @@ func (rpcp *RPCProvider) SetupProviderEndpoints(rpcProviderEndpoints []*lavasess
 	wg.Add(parallelJobs)
 	disabledEndpoints := make(chan *lavasession.RPCProviderEndpoint, parallelJobs)
 	// validate static spec configuration is used only on a single chain setup.
-	chainIds := make(map[string]struct{})
+	providerLoadManagersPerChain := make(map[string]*ProviderLoadManager)
```

Comment:
we have a chain common setup function, use that one to add an asset that is for a chain

Reply:
Fixed in [810db13](https://github.com/lavanet/lava/pull/1720/commits/810db1333787e581940cf5314e27bd9aa1f5cab5)